### DISCUSSION TITLE
ISTA Netherlands

### DISCUSSION THREAD

**Discussion by serot23** [Ludy87/ecotrend-ista#188] [Ideas]
Hi there,

I am not that technical in the sense that i understand how pulls etc works in github and how to make my own version by using an api or something. But in the Netherlands we also have ISTA (unfortunately) and i would like to inplement it in HA, but i can't use this integration, which is a bummer.  Is is something simple as adding countries to your integration or is it a lot of work to add extra apies or something? 

Kind regards Serot23

**Comment by MiroG225** [comment]
I'm also not sure it's possible, but I'd like to use this for https://ista24.cz/

**Comment by TychoWerner** [comment]
Would love to hear this too, where it shows more URL aswell
https://github.com/Ludy87/ecotrend-ista/blob/main/custom_components/ecotrend_ista/strings.json

**Reply by LukTyn** [reply]
Simply add "czech": "https://ecotrend.ista.cz" doesn't do anything. 
And I guess there's user name not e-mail for login in our country. 
What else should I do?

**Comment by sems** [comment]
Sadly to see that there is still no update on this feature request.
